docs/buildingpex.rst

```diff
@@ -63,6 +64,22 @@ You can then import and manipulate modules like you would otherwise::
     >>> psutil.__path__
     ['/private/var/folders/4d/9tz0cd5n2n7947xs21gspsxc0000gp/T/tmpYuGpFW/.deps/psutil-2.0.0-py2.6-macosx-10.4-x86_64.egg/psutil']
 
+Specifying source directories
+-----------------------------
+
+Source directories are source distributions typically built with ``python setup.py sdist``.
```

Comment:
This wording is slightly backward.  Instead it should be something like

"Python source distributions are source directories with a properly defined setup.py."